packages/grid/x-data-grid/src/components/panel/filterPanel/GridFilterForm.tsx

```diff
@@ -142,7 +140,7 @@ const GridFilterFormRoot = styled('div', {
   padding: theme.spacing(1),
 }));
 
-const FilterFormDeleteIcon = styled(FormControl, {
+const FilterFormDeleteIcon = styled('div', {
```

Comment:
I wonder what functionality eas being achieved by this component (FormControl) 🤔

Reply:
Not sure about the motivation behind it, but `FormControl` is passed to the `as` prop down below anyway 🤷🏻‍♂️